documentation/doxygen/print_roofit_pyz_doctrings.py

```diff
@@ -0,0 +1,124 @@
+# -------------------------------------------------------------------------------
+#  Author: Jonas Rembser <jonas.rembser@cern.ch> CERN
+# -------------------------------------------------------------------------------
+
+################################################################################
+# Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+# Creates the doxygen documentation for the RooFit pythonizations, which includes:
+#   - a PyROOT box for each pythonized class or member function
+#   - a separate page for the RoofitPythonizations group where all the RooFit
+#     pythonization documentation is aggregated
+
+import inspect
+
+
+def write_pyroot_block_for_class(klass):
+
+    if not hasattr(klass, "_doxygen"):
+        return
+
+    print("\class " + klass.__name__)
+    print("\\brief \parblock \endparblock")
+    print("\htmlonly")
+    print('<div class="pyrootbox">')
+    print("\endhtmlonly")
+    print("## PyROOT")
+
+    print(inspect.cleandoc(klass._doxygen))
+
+    print("\htmlonly")
+    print("</div>")
+    print("\endhtmlonly")
+    print("")
+
+
+def write_pyroot_block_for_member_func(func):
+
+    if not hasattr(func, "_doxygen") or not hasattr(func, "_cpp_signature"):
+        return
+
+    print("\\fn " + func._cpp_signature)
+    print("\\brief \parblock \endparblock")
+    print("\htmlonly")
+    print('<div class="pyrootbox">')
+    print("\endhtmlonly")
+    print("## PyROOT")
+
+    print(inspect.cleandoc(func._doxygen))
+
+    print("\htmlonly")
+    print("</div>")
+    print("\endhtmlonly")
+    print("")
+
+
+if __name__ == "__main__":
+
+    import ROOT.pythonization as pyz
+
+    # Fill separate RooFit pythonization page, starting with the introduction and table of contents...
```

Comment:
Can we have two helper functions here, one for the generation of the RooFit page and another one for the addition of the PyROOT block for the RooFit classes?

Reply:
Okay!